README.md

```diff
@@ -11,7 +11,7 @@ are primarily targeted for the Mbed (SDP-K1) platform, however they can easily
 target other platforms as well (see [Understanding No-OS and
 Platform Drivers](https://www.analog.com/en/analog-dialogue/articles/understanding-and-using-the-no-os-and-platform-drivers.html)).
 
-For more information about Precision Converters software support see the [ADI Wiki](https://wiki.analog.com/resources/tools-software/product-support-software).
+For more information about Precision Converters software support see the [Documentation for the Precision Converters Firmware](https://analogdevicesinc.github.io/precision-converters-firmware/index.html).
```

Comment:
nit: Instead of saying 'Precision Converters software support' should we explicitly say something like this - 
https://github.com/analogdevicesinc/no-OS#:~:text=Code%20documentation%20is%20automatically%20generated%20using%20Doxygen%20tool%2C%20available%20at%20no%2DOS%20Github%20Pages.

For additional 'support', we're already pointing them to EZone, in the documentation.